src/openrct2/cmdline/RootCommands.cpp

```diff
@@ -459,6 +459,10 @@ static void PrintVersion()
     Console::WriteLine();
     Console::WriteFormat("Minimum park file version: %d", OpenRCT2::PARK_FILE_MIN_VERSION);
     Console::WriteLine();
+#ifdef USE_BREAKPAD
+    Console::WriteFormat("With breakpad support enabled");
+    Console::WriteLine();
+#endif
```

Comment:
Perhaps also say that it isn't enabled when it isn't enabled.

Reply:
done